source/releasenotes/2024-04-24-php-8-3-recommended.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "PHP 8.3 is now recommended on Pantheon"
+published_date: "2024-04-24"
+categories: [infrastructure]
+---
+
+Following our announcement of the New Relic PHP agent's compatibility with PHP 8.3, we are [now recommending PHP 8.3](https://docs.pantheon.io/guides/php) for WordPress & Drupal applications on the platform. PHP 8.1 and 8.2 are still recommended as well.
```

Comment:
```suggestion
Following our announcement of the New Relic PHP agent's compatibility with PHP 8.3, we are [now recommending PHP 8.3](/guides/php) for WordPress & Drupal applications on the platform. PHP 8.1 and 8.2 are still marked as "recommended" as well because it is common for plugins and modules in the ecosystem of still have incompatibilities or other issues with newer versions of PHP.

[Try out your site with a newer version of PHP by updating your `pantheon.yml` file](/guides/php/php-versions#configure-your-php-version) in a Dev or [Multidev](/guides/multidev) environment. 
```